### PR TITLE
Feature/full control of img attrs

### LINKED ISSUE #7: title attribute being parsed with twemoji.js
I'm using twemoji.js on the front end for some preview features. I'm using twemoji.rb for backend rendering. If I take these steps I run into issues:

  1. If I backend render an img tag with Twemoji.parse that includes something like the deafult title `title = ":heart_eyes:"`
  1.  edit that text (which includes the `` tag) in a wysiwyg editor on the front end after backend rendering
  1. `twemoji.parse` in the frontend for preview the title attribute value is replaced for another img tag

:(

We can fix this a number of ways in our own app, but if we made one of three possible easy modifications to twemoji.rb it would be a non-issue. 

  1. change title to be the shortname without the `:` so `heart_eyes` instead of `:heart_eyes:`
  1. remove title attribute from default (adding when needed)
  1. make all the attributes in the image tag come from an options hash like `options[:default_img_attrs]` or make `options[:img_attrs]` that default.

I would be happy to code whatever solution you prefer.

Thanks!

### PR DESCRIPTION
**Pull request by JuanitoFatas**
Background #7 #9 

This pull request adds full control of rendered img tag attributes. You can even apply a Proc-like object to do things about attribute's value.

- [x] rebase with master
- [x] CHANGELOG entry [Rendered CHANGELOG.md](https://github.com/jollygoodcode/twemoji/blob/58c783cf2299a8d8116a755861002ee99920f163/CHANGELOG.md)
- [x] Fix README.md [Rendered README.md](https://github.com/jollygoodcode/twemoji/blob/5f634be63f36ac917b8d51e5f359077b02bcc518/README.md)
- [x] When release, need a MAJOR version bump 

Fixes #7

### CODE REVIEW

File: lib/twemoji.rb [hunk 1]
```diff
@@ -206,12 +204,20 @@ def self.filter_emojis(content)
...
     # @private
     def self.img_tag(name)
-      unicode = render_unicode(name)
-      if options[:img_attr]
-        "<img class='#{options[:class_name]}' draggable='false' title='#{name}' alt='#{unicode}' #{options[:img_attr]} src='#{emoji_url(name)}'>"
-      else
-        "<img class='#{options[:class_name]}' draggable='false' title='#{name}' alt='#{unicode}' src='#{emoji_url(name)}'>"
+      # set default attributes and override from img_attrs
+      attributes = {
+        draggable: "false",
```
Comment (JuanitoFatas) [inline lib/twemoji.rb]: @winston I think it is better we only provide default for `title`, `alt`, and `src`, other attributes we leave the user to decide, so I want to remove this `draggble: false`, :ok_hand:  ?

Comment (choonkeat) [inline lib/twemoji.rb]: based on https://github.com/twitter/twemoji i'd leave it in unless there's a reason to take it out

Comment (JuanitoFatas) [inline lib/twemoji.rb]: Got it. Thanks @choonkeat!

**Comment by JuanitoFatas** [comment]
@winston @choonkeat Please take a look :eyes:

**Comment by JuanitoFatas** [comment]
> MAJOR version when you make incompatible API changes,
> MINOR version when you add functionality in a backwards-compatible manner, and
> http://semver.org/

Does this API change count as a MAJOR version bump or MINOR?

**Comment by winston** [comment]
LGTM. And yes, this is a breaking change.

**Pull request merged**